app/routes/bekk/routes.py

```diff
@@ -0,0 +1,177 @@
+"""PC est magique - Bekk Gallery Routes"""
+
+import flask
+from flask_babel import _
+
+from app import context, db
+from app.routes.bekk import bp, forms
+from app.utils import typing
+from app.models import (
+    Bekk,
+    PermissionScope,
+    PermissionType,
+    GlobalSetting,
+    PCeen,
+    Role,
+    Permission,
+)
+
+import fitz
+import os
+import shutil
+
+
+@bp.route("", methods=["GET", "POST"])
+@bp.route("/", methods=["GET", "POST"])
+# @context.permission_only(PermissionType.read, PermissionScope.bekk)
+def main() -> typing.RouteReturn:
+    """Bekk module main page"""
+
+    bekks = Bekk.query.order_by(Bekk.date.desc()).all()
+    promos = []
+    [promos.append(bekk.promo) for bekk in bekks if bekk.promo not in promos]
+    promos.sort(reverse=True)
+
+    promo = flask.request.args.get("promo")
+    if promo == None:
```

Comment:
plutôt utiliser `is` avec None : https://stackoverflow.com/a/14247383

```suggestion
    if promo is None:
```